website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -60,38 +67,31 @@ We will use the `v1.3.0` release version in this example.
    by running the
    [deploy-manual](https://github.com/operator-framework/operator-sdk/actions/workflows/deploy-manual.yml)
    GitHub action.  After the image is built, check the security scan
-   results under `Child Manifests`.
-1. A release branch must be created. If you have the proper permissions,
-   you can do this by running the following, assuming the upstream SDK
+   results under `Child Manifests`. Merge the resulting pull requests to update the links.
+1. Make sure that the list of supported OLM versions stated in the [Overview][overview] section of SDK docs is updated. If a new version of OLM needs to be officially supported, follow the steps in [updating OLM bindata](#updating-olm-bindata) section.
```

Comment:
The OLM bindata and the child manifests seems out of place. I think this should be part of the release items in the milestone instead. Because they take a while and it sucks to have to wait for these to even start a build. 

I'd rather we add these as standard release tasks in the milestones.

Reply:
Another option would be to write a script that updates the olm bindata and run it as a pre-release step. We already have a GitHub action that verifies if bindata is out of date, having automation to update it would be more helpful. But definitely a follow up task we could create an issue for.